src/assets/stylesheets/components/_form-input-group.scss

```diff
@@ -0,0 +1,20 @@
+.form-input-group {
```

Comment:
These styles shouldn't be a part of the `component` as these classes are defined in a `screen` 🤔

Reply:
That was my sloppy. Fixed in 19fd8543a38982106c774680efb5c9e69a454d2a